pkg/certificate/certificate.go

```diff
@@ -126,3 +131,34 @@ func GetCertificateSANs(host string, alternativeNames []string) []string {
 
 	return certSANS
 }
+
+func RenewAll(st *state.State) error {
+	return st.RunTaskOnControlPlane(func(ctx *state.State, node *kubeoneapi.HostConfig, _ executor.Interface) error {
+		logger := ctx.Logger.WithField("node", node.PublicAddress)
+		logger.Infoln("Renew certificates...")
+
+		_, _, err := ctx.Runner.RunRaw("sudo kubeadm certs renew all")
+		if err != nil {
+			return fail.SSH(err, "renewing certificates")
+		}
+
+		pods := []string{
+			"etcd",
+			"kube-apiserver",
+			"kube-controller-manager",
+			"kube-scheduler",
+		}
+
+		for _, pod := range pods {
+			logger.Infof("Restarting %s pod...", pod)
+			_, _, err := ctx.Runner.Run(scripts.RestartPodCrictlTemplate, runner.TemplateVariables{
+				"NAME": pod,
+			})
+			if err != nil {
+				return fail.SSH(err, "renewing certificates")
+			}
+		}
+
+		return nil
+	}, state.RunParallel)
```

Comment:
I'd run this sequentially because it involves restarting pods, so restarting in parallel might affect the control plane availability.